src/blocks/stars.css

```
@@ -0,0 +1,21 @@
+.box {
+  display: flex;
+  align-items: center ;
+  justify-content: space-between ;
+}
+
+.stars__star {
+  width: 16px ;
+  height: 16px ;
+  padding-right: 20px ;
+  background: url("../images/star.svg") 2px 2px /13px no-repeat ;
+}
+
+.stars {
+  display: flex ;
+  align-items: center ;
+}
+
+.stars--4 :nth-child(-n+4) {
```

```suggestion
.stars--1 :nth-child(-n+1) {
.stars--2 :nth-child(-n+2) {
.stars--3 :nth-child(-n+3) {
.stars--4 :nth-child(-n+4) {
.stars--5 :nth-child(-n+5) {
```
add all numbers from stars task